lxc/remote.go

```diff
@@ -111,6 +115,10 @@ func addServer(config *lxd.Config, server string, addr string) error {
 		return nil
 	}
 
+	if acceptCert {
+		c.AcceptCert = true
```

Comment:
Can we make this an argument to UserAuthServerCert vs. a member on the Client struct? I think it's only relevant for a single call to the function, vs. all subsequent calls.